plugins/paymethod/paypal/locale/cs_CZ/locale.xml

```diff
@@ -4,8 +4,8 @@
 <!--
   * plugins/paymethod/paypal/locale/cs_CZ/locale.xml
   *
-  * Copyright (c) 2014-2016 Simon Fraser University Library
-  * Copyright (c) 2006-2014 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2006-2009 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
```

Comment:
I'm nervous about rolling back the dates for non-team-members; best to leave this as it was.